osgp/protocol-adapter-iec60870/osgp-iec60870/src/main/java/org/opensmartgridplatform/iec60870/Iec60870Server.java

```diff
@@ -74,6 +74,27 @@ public void setProcessImage(final Map<Integer, InformationElement[][]> processIm
     this.processImage = processImage;
   }
 
+  /**
+   * Initializes the value for a specific information object address
+   *
+   * <p>Contrary to the {@link Iec60870Server#updateInformationObject(int,
+   * Iec60870InformationObjectType, Object)} method, no event is sent to the controlling station.
+   *
+   * @param informationObjectAddress
+   * @param informationObjectType
+   * @param value
+   */
+  public void initInformationObject(
```

Comment:
Does it make sense to initialize the information objects separately (could be if there is only one in the image)?
Otherwise it would be better to replace this by, or ad an initProcessImage method that takes the `Iec60870ProcessImage` (may be what setProcessImage already provides). Initialization gives the impression that it has to do with set up which for me stronger relates to the entire image than to the separate parts.
Creating the image in the ProcessImageSteps, and then not setting it on the server, but initializing the seperate information objects where they will together form the image again feels like taking a detour. Or is there a reason to take this seemingly longer road?